trivy@2 Unexpected token Agent.TempDirectory

#### Describe the bug

SyntaxError: Unexpected token ?

#### Agent details

* Type: Self-Hosted Agent
* OS: Linux X64
* Version: 3.218.0
```
Starting: Initialize job
Agent name: 'ado-agent-scaledjob-ukdxl-java-gf4jj-7hl4q'
Agent machine name: 'ado-agent-scaledjob-ukdxl-java-gf4jj-7hl4q'
Current agent version: '3.218.0'
Agent running as: 'root'
Prepare build directory.
Set build variables.
Download all required tasks.
Downloading task: NodeTool (0.247.1)
Downloading task: UseNode (1.248.1)
Downloading task: Bash (3.250.1)
Downloading task: Cache (2.198.0)
Downloading task: trivy (2.2.83)
Downloading task: PublishTestResults (2.254.0)
Checking job knob settings.
   Knob: DockerActionRetries = true Source: $(VSTSAGENT_DOCKER_ACTION_RETRIES) 
   Knob: AgentToolsDirectory = /azp/tools Source: ${AGENT_TOOLSDIRECTORY} 
   Knob: ContinueAfterCancelProcessTreeKillAttempt = true Source: $(VSTSAGENT_CONTINUE_AFTER_CANCEL_PROCESSTREEKILL_ATTEMPT) 
Finished checking job knob settings.
Start tracking orphan processes.
```

#### Task Version

Azure Extention: 1.16.1

#### Task Inputs

```yaml
              - task: trivy@2
                name: TrivyCurrent
                displayName: "Trivy Filesystem and Misconfiguration Scan"
                inputs:
                  version: $(Profile.TrivyVersion)
                  type: 'filesystem'
                  target: '.'
                  scanners: 'misconfig,vuln,license,secret'
                  severities: 'HIGH,CRITICAL'
                  ignoreScanErrors: true
                  reports: 'github, html, junit'
                  publish: true
```

#### Error message

<!-- Details of the error message that you are seeing -->
```log
Starting: Trivy Filesystem and Misconfiguration Scan
==============================================================================
Task         : Trivy
Description  : Trivy is the world's most popular open source vulnerability and misconfiguration scanner. It is reliable, fast, extremely easy to use, and it works wherever you need it.
Version      : 2.2.83
Author       : Aqua Security
Help         : https://github.com/aquasecurity/trivy-azure-pipelines-task/docs/trivyv2.md
==============================================================================
##[error]Unhandled: Unexpected token ?
##[error]/azp/_work/_tasks/trivy_8f9cb13f-f551-439c-83e4-fac6801c3fab/2.2.83/dist/runner.js:13
const agentTmp = task.getVariable('Agent.TempDirectory') ?? os_1.default.tmpdir();
                                                          ^

SyntaxError: Unexpected token ?
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/azp/_work/_tasks/trivy_8f9cb13f-f551-439c-83e4-fac6801c3fab/2.2.83/dist/index.js:9:18)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
```

When running:
```
              - bash: |
                  env
                displayName: "Environment Variables"
```
Return is:
```
AGENT_TEMPDIRECTORY=/azp/_work/_temp
```

The whole trace of the task:
```
##[debug]loading INPUT_SCANNERS
##[debug]loading INPUT_SEVERITIES
##[debug]loading INPUT_IGNOREUNFIXED
##[debug]loading INPUT_IGNORESCANERRORS
##[debug]loading INPUT_REPORTS
##[debug]loading INPUT_PUBLISH
##[debug]loading ENDPOINT_AUTH_SYSTEMVSSCONNECTION
##[debug]loading ENDPOINT_AUTH_SCHEME_SYSTEMVSSCONNECTION
##[debug]loading ENDPOINT_AUTH_PARAMETER_SYSTEMVSSCONNECTION_ACCESSTOKEN
##[debug]loading SECRET_SYSTEM_ACCESSTOKEN
##[debug]loaded 14
##[debug]Agent.ProxyUrl=undefined
##[debug]Agent.CAInfo=undefined
##[debug]Agent.ClientCert=undefined
##[debug]Agent.SkipCertValidation=undefined
##[debug]task result: Failed
##[error]Unhandled: Unexpected token ?
##[debug]Processed: ##vso[task.issue type=error;source=TaskInternal;]Unhandled: Unexpected token ?
##[debug]Processed: ##vso[task.complete result=Failed;]Unhandled: Unexpected token ?
##[error]/azp/_work/_tasks/trivy_8f9cb13f-f551-439c-83e4-fac6801c3fab/2.2.83/dist/runner.js:13
const agentTmp = task.getVariable('Agent.TempDirectory') ?? os_1.default.tmpdir();
                                                          ^

SyntaxError: Unexpected token ?
    at Module._compile (internal/modules/cjs/loader.js:723:23)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:692:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/azp/_work/_tasks/trivy_8f9cb13f-f551-439c-83e4-fac6801c3fab/2.2.83/dist/index.js:9:18)
    at Module._compile (internal/modules/cjs/loader.js:778:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
##[debug]Processed: ##vso[task.issue type=error;source=TaskInternal;]/azp/_work/_tasks/trivy_8f9cb13f-f551-439c-83e4-fac6801c3fab/2.2.83/dist/runner.js:13%0Aconst agentTmp = task.getVariable('Agent.TempDirectory') ?? os_1.default.tmpdir();%0A                                                          ^%0A%0ASyntaxError: Unexpected token ?%0A    at Module._compile (internal/modules/cjs/loader.js:723:23)%0A    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)%0A    at Module.load (internal/modules/cjs/loader.js:653:32)%0A    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)%0A    at Function.Module._load (internal/modules/cjs/loader.js:585:3)%0A    at Module.require (internal/modules/cjs/loader.js:692:17)%0A    at require (internal/modules/cjs/helpers.js:25:18)%0A    at Object.<anonymous> (/azp/_work/_tasks/trivy_8f9cb13f-f551-439c-83e4-fac6801c3fab/2.2.83/dist/index.js:9:18)%0A    at Module._compile (internal/modules/cjs/loader.js:778:30)%0A    at Object.Module._extensions..js (internal/modules/cjs/loader.js:789:10)
Finishing: Trivy Scan
```

--- a/trivy-task/trivyV2/task.json
+++ b/trivy-task/trivyV2/task.json
@@ -10,7 +10,7 @@
   "author": "Aqua Security",
   "version": {
     "Major": 2,
-    "Minor": 2,
+    "Minor": 3,
     "Patch": 0
   },
   "instanceNameFormat": "Echo trivy $(version)",
@@ -211,7 +211,7 @@
     "Node20_1": {
       "target": "dist/index.js"
     },
-    "Node10": {
+    "Node16": {
       "target": "dist/index.js"
     }
   }

